tests/__init__.py

```diff
@@ -61,6 +61,7 @@ def mkosi(
         user: Optional[int] = None,
         group: Optional[int] = None,
         check: bool = True,
+        env: Optional[Mapping[str, str]] = None,
```

Comment:
Couldn't this be made a bit tighter?
```suggestion
        env: Mapping[str, str] = os.environ,
```
This would also make the fallback clear in the signature without having to scroll through the code.